File: src/main/java/training/core/LargestNumber/LargestNumber.java
package training.core.LargestNumber;

class Solution {
    int largestNumber(int n) {
        int result = 9;
        int i = 1;
        while (i < n) {
            result += 9 * Math.pow(10.0, i);
            i++;
        }
        return result;
    }
}

File: src/main/java/training/core/LargestNumber/LargestNumber.cs
namespace training.core.LargestNumber
{
    using System;
    public class Solution
    {
        public int largestNumber(int n)
        {
            var result = 9;
            for (var i = 1; i < n; i++)
                result += 9 * Convert.ToInt32(Math.Pow(10.0, Convert.ToDouble(i)));
            return result;
        }
    }
}
